.github/workflows/ci.yml

```diff
@@ -4,7 +4,7 @@ on:
     branches:
       - main
   pull_request:
-    types: [labeled, synchronize]
+    types: [opened, labeled, synchronize]
```

Comment:
Do we still need `synchronize` as a type here?

Reply:
I thought we did, but perhaps not? 🤔 Without it this action should still run once a PR has been opened. I'll trying removing it…

Reply:
No, it is still needed. Without `synchronize` pushes to the branch no longer trigger the action.

Reply:
ok great - thanks for checking!